packages/cxl-ui/src/components/cxl-jw-player/mixins/TranscriptMixin.js

```diff
@@ -84,7 +84,13 @@ export function TranscriptMixin(BaseClass) {
         });
       }
 
-      return tracks;
+      const filteredTracks = tracks.map((track) => {
+        track.data.text = track.data.text.replace("<v ->", "").replace("</v>", "");
```

Comment:
Idk this looks weird, like it's unclear what the real root problem is.

Reply:
HTML seems to be coming directly from the response.

![image](https://github.com/user-attachments/assets/c2c19be0-469c-45ae-91d5-26470d4c1392)

Reply:
`<v>` seems to be a WebVTT Voice tag (TIL!) https://developer.mozilla.org/en-US/docs/Web/API/WebVTT_API/Web_Video_Text_Tracks_Format#voice_tag_

https://developer.mozilla.org/en-US/docs/Web/API/WebVTT_API

So it seems like legit output, although

* unclear why it's sent, especially with `-` value.. does it mean voice owner unknown, or what's generating this?
* I think our implementation needs to be able to render WebVTT tags, not try to remove them?

Reply:
> unclear why it's sent, especially with - value.. does it mean voice owner unknown, or what's generating this?

This is coming directly from JW Player, so I believe whichever software we used to generate these captions is responsible.

> I think our implementation needs to be able to render WebVTT tags, not try to remove them?

We aren't using the WebVTT API. We use this library to parse the VTT files, so that we can render them outside of the video: https://github.com/gsantiago/subtitle.js

The WebVTT API, and the tags associated with it seem to only be useful when adding captions to the actual video itself. Since we are rendering the captions outside of the video, I'm not sure how helpful they are.

Reply:
@anoblet what did we do differently in the old code? I recall we had some JS plugins for JWP.